modular_skyrat/modules/tribal_extended/code/ammo/caseless/arrow.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/ammo_casing/caseless/arrow/wood
+	name = "wooden arrow"
+	desc = "An arrow made of wood, typically fired from a bow."
+	icon = 'modular_skyrat/modules/tribal_extended/icons/ammo.dmi'
+	icon_state = "arrow"
+	inhand_icon_state = "arrow"
+
+/obj/item/ammo_casing/caseless/arrow/ash
+	name = "ashen arrow"
+	desc = "An arrow made from wood, hardened by fire"
```

Comment:
Ashen arrow's description is missing a comma.

Reply:
PERIOD I MEAN

Reply:
Done